src/AccessibilityInsights.Extensions.AzureDevOps/Models/IssueResult.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using AccessibilityInsights.Extensions.Interfaces.IssueReporting;
+using System;
+
+namespace AccessibilityInsights.Extensions.AzureDevOps.Models
+{
+    /// <summary>
+    /// Contains data to be displayed in AI-Win after an issue is filed
+    /// </summary>
+    public class IssueResult : IIssueResult
```

Comment:
something like this would be helpful to encapsulate the same entities in Core.Results.RuleResult.cs but I'm not sure if it can be shared